docs/developers_guide/api.rst

```diff
@@ -72,6 +72,7 @@ Ocean tasks
    ClimatologyMapSose
    ClimatologyMapArgoTemperature
    ClimatologyMapArgoSalinity
+   ClimatologyMapWaves
```

Comment:
I'm seeing:
```
/gpfs/fs1/home/ac.xylar/mpas-work/MPAS-Analysis/waves/docs/developers_guide/api.rst:61: WARNING: autosummary: failed to import ClimatologyMapWaves.
Possible hints:
* ModuleNotFoundError: No module named 'mpas_analysis.ocean.ClimatologyMapWaves'
* KeyError: 'ClimatologyMapWaves'
* ModuleNotFoundError: No module named 'ClimatologyMapWaves'
* ValueError: not enough values to unpack (expected 2, got 1)
* AttributeError: module 'mpas_analysis.ocean' has no attribute 'ClimatologyMapWaves'
```
I don't yet understand why.

Reply:
Okay, ignore this.  It seems fine on re-testing.  Maybe just needed a `make clean`.